shipment_advice/models/shipment_advice.py

```diff
@@ -383,7 +384,7 @@ def _validate_picking(self, picking, backorder_policy="create_backorder"):
                     wiz.pick_ids = picking
                     wiz.with_context(button_validate_picking_ids=picking.ids).process()
                 elif not picking._check_backorder():
-                    picking._action_done()
+                    picking.with_context(skip_backorder=True).button_validate()
```

Comment:
Should we also set `skip_immediate` context key? Two actions can be returned by submethod `_pre_action_done_hook`, and we can imagine other modules adding their own wizards/actions to pop here.

Also to be on the safe-side if we want to use `button_validate`, is to check the state of the picking right after (raising a `UserError` catched below?)